Automattic/newspack-blocks
# feat(modal-checkout): update thank you page with new newspack ui

### All Submissions:






### Changes proposed in this Pull Request:



Part of `1206197591054328/1206405274735557/f`

This applies the newspack ui to the modal checkout thank you page and prepares the rest of the modal checkout to be updated to the new ui. As a consequence this breaks the styling of the modal checkout and billing forms a bit, but I will work on those in a follow up PR so this one doesn't get too big:

Checkout form is expected to look like this:
![Screenshot 2024-01-30 at 11 45 15](https://github.com/Automattic/newspack-blocks/assets/17905991/df96cc5e-dac6-400c-915f-d91bde6fcf64)

Billing info form is expected to look like this:
![Screenshot 2024-01-30 at 11 45 25](https://github.com/Automattic/newspack-blocks/assets/17905991/6dc217a2-6c94-4a98-b7d8-8a8afce17a3c)

### How to test the changes in this Pull Request:

1. With newspack plugin on `feat/update-checkout-block-thankyou-ui` branch, trigger modal checkout via donate button
2. Complete checkout until you see the thank you page
3. Confirm the thankyou modal looks as expected with the udpated ui

![Screenshot 2024-01-25 at 14 31 24](https://github.com/Automattic/newspack-blocks/assets/17905991/0226b79b-5fe5-4688-8bf9-928b5da71fa2)

4. Switch to `master` on newspack plugin and repeat the above steps
5. Confirm nothing appears to be broken (The thank you page should not have the new styles applied and just be a generic transaction summary)


### Other information:
